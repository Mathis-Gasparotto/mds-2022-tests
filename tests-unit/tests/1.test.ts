import { expect, test } from 'vitest'
import { filterAndSortUsers, User } from '../1'

const users: User[] = [
  { age: 17, name: 'Bob' },
  { age: 25, name: 'Alice' },
  { age: 30, name: 'Charlie' }
]

test('Function should filter array by minor users, and sort by name', () => {
  const actual = filterAndSortUsers(users)

  expect(actual).toMatchSnapshot()
})
