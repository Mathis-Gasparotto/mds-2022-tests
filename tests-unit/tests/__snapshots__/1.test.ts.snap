// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Function should filter array by minor users, and sort by name 1`] = `
[
  {
    "age": 30,
    "name": "Charlie",
  },
  {
    "age": 25,
    "name": "Alice",
  },
]
`;
